library/src/commonMain/kotlin/com/lagradost/cloudstream3/utils/SubtitleHelper.kt

```diff
@@ -154,19 +157,23 @@ object SubtitleHelper {
         return languages.getOrNull(index)
     }
 
-    // @Deprecated(
-    //     "Default language code changed to IETF BCP 47 tag",
-    //     replaceWith = ReplaceWith("fromTagToLanguageName(input)"))
+    @Deprecated(
+        "Default language code changed to IETF BCP 47 tag",
+        replaceWith = ReplaceWith("fromTagToLanguageName(input)"),
```

Comment:
Sorry, I think the correct replacement is `fromTagToEnglishLanguageName` instead of `fromTagToLanguageName` (which translates the name)
```suggestion
        replaceWith = ReplaceWith("fromTagToEnglishLanguageName(input)"),
```